src/main/webapp/WEB-INF/tags/navBarBrand.tag

```diff
@@ -0,0 +1,5 @@
+<%@ tag description="TEAMMATES logo brand"%>
+
+<a class="navbar-brand" href="/">
+    <img height="100%" src="../images/teammateslogo.jpg" alt="TEAMMATES[Logo] - Online Peer Feedback/Evaluation System for Student Team Projects">
```

Comment:
The home page and the action pages have different request URLs, so the relative path to the image would be different - the home page should use `images/teammateslogo.jpg` while the other pages should use `../images/teammateslogo.jpg`. Now `../images` works for the home page only because one level up from the root still resolves to the root. One workaround is to use `/images/teammateslogo.jpg` instead, which is root-relative and so would be correct for all pages.